sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -162,11 +162,13 @@ void mask_rom_boot(void) {
     // if (!final_jump_to_rom_ext(current_rom_ext_manifest)) { // Hardened Jump
     // Module
     if (true) {
+      uintptr_t entry_point;
```

Comment:
Declare this as `romextimage_entry_point *entry_point;` ?

Reply:
Please see above.